File: app/src/main/java/com/gtn/abhis/crackit4/Login_Page.java
package com.gtn.abhis.crackit4;

import android.app.Application;
import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class Login_Page extends AppCompatActivity {

    //Not a login page. Home Page

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login__page);

    }

    public void easypressed(View view)
    {
        Intent intent = new Intent(Login_Page.this,MainActivity.class).putExtra("mode", "easy");
        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
        startActivity(intent);
    }

    public void hardpressed(View view)
    {
        Intent intent = new Intent(Login_Page.this,MainActivity.class).putExtra("mode", "easy");
        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
        startActivity(intent);
    }

    public void about(View view)
    {
        Intent intent = new Intent(Login_Page.this,AboutMe.class);
        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
        startActivity(intent);
    }

    public void howtoplay(View view)
    {
        Intent intent = new Intent(Login_Page.this,HowToPlay.class);
        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
        startActivity(intent);
    }

    public void quitit(View view)
    {
        System.exit(1);
    }
}

File: app/src/main/java/com/gtn/abhis/crackit4/LoginPage.kt
package com.gtn.abhis.crackit4

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class LoginPage : AppCompatActivity() {
    //Not a login page. Home Page
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login_page)
    }

    fun easypressed(view: View?) {
        val intent = Intent(this, MainActivity::class.java).putExtra("mode", "easy")
        intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
        startActivity(intent)
    }

    fun hardpressed(view: View?) {
        val intent = Intent(this, MainActivity::class.java).putExtra("mode", "easy")
        intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
        startActivity(intent)
    }

    fun about(view: View?) {
        val intent = Intent(this, AboutMe::class.java)
        intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
        startActivity(intent)
    }

    fun howtoplay(view: View?) {
        val intent = Intent(this, HowToPlay::class.java)
        intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
        startActivity(intent)
    }

    fun quitit(view: View?) {
        System.exit(1)
    }
}